crypto/dilithium/pqcrystals_dilithium_ref_common/sign.h

```diff
@@ -5,48 +5,48 @@
 #include <stdint.h>
 #include "params.h"
 
-int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk);
+int mldsa_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk);
```

Comment:
Missed renaming the header file include guard on line 1

Reply:
[2bbdfc9](https://github.com/aws/aws-lc/pull/2072/commits/2bbdfc975539e79985135f515caa45399229d1f6)